worklist/constants.py

```diff
@@ -9,7 +9,10 @@
 # Constant for mapping article status to a
 # number for storing purposes
 ARTICLE_STATUS_TO_NUMBER_MAPPING = \
-    {'open': 0, 'closed': 1, 'in_progress': 2}
+    {'open': 0, 'closed': 1, 'claimed': 2}
+
+ARTICLE_NUMBER_TO_STATUS_MAPPING = \
```

Comment:
Nit: I think I'd just invert the dict above so the mapping is expressed in a single place only:

```python
ARTICLE_NUMBER_TO_STATUS_MAPPING = {
  number : status for status, number in ARTICLE_STATUS_TO_NUMBER_MAPPING.items()
}
```

Reply:
Done.